Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -40,7 +40,9 @@ ActsExamples::SeedingAlgorithm::SeedingAlgorithm(
   }
 
   if (m_cfg.gridConfig.rMax != m_cfg.seedFinderConfig.rMax) {
-    throw std::invalid_argument("Inconsistent config rMax");
+    ACTS_WARNING(
```

Comment:
One consideration is that in **principle** we want the CI and automated tests to fail if there's any log message of level WARNING or above (we currently have some exceptions but in principle we don't want to add more).

Could we make this an extra flag like `m_cfg.allowSeparateRMax` and if that is true, no output happens and if false we print this warning (or go back to the exception)?

Reply:
I think then we could just remove the warning altogether, this warning/exception only helps someone who didn't expect that this should be the same (in most cases), if there's a parameter you'd need to switch to find out, you may as well just check your configuration.

@LuisFelipeCoelho, maybe you can explain under which circumstances a higher rMax in the seedFinder than in the Grid makes sense, given that the Grid's rMax removes all SP outside of this area.

Reply:
The rMax in the grid is only used to calculate deltaPhi of the Grid in `SpacePointGrid.ipp`. Then, rMax in the seedFinder is used in `BinnedSPGroup.ipp` to remove all SP outside the region defined by rMax.

In Athena, rMax is only used to calculate deltaPhi of the Grid and there is no r limit to SPs that are used by the Seedfinder. I am not sure why Athena does that, but If we set them equal and limit the SPs we are removing measurements that are used by Athena, and if we increase the grid rMax we change the grid and the seeds are different. 

I don't mind adding an extra flag `m_cfg.allowSeparateRMax`

Reply:
sounds good, let's add the additional flag m_cfg.allowSeparateRMax and only throw the warning if that's not True.